buildsystem/spack/deception/spack.yaml

```
@@ -24,9 +24,9 @@ spack:
     magma:
       require: '@2.6.2'
     coinhsl:
-      require: '@2019.05.21'
+      require: '@2023.11.17'
```

```suggestion
      require: '@2023.11.17+metis'
```

@pelesh per your suggestion

---

Or is it sufficient just to build Ipopt with metis... Or do we need both? Should the spack package force `coinhsl+metis` if `ipopt+metis` for these later versions?

---

Ipopt does not depend on Metis, only HSL does. I am not sure what `ipopt+metis` is supposed to do tbh.

Forcing `coinhsl+metis` is the right thing to do imho.